vllm/worker/worker.py

```diff
@@ -83,7 +86,8 @@ def __init__(
         # Uninitialized cache engine. Will be initialized by
         # initialize_cache.
         self.cache_engine: CacheEngine
-        self.gpu_cache: List[torch.Tensor]
+        # Initialize gpu_cache as embedding models don't initialize kv_caches
+        self.gpu_cache: List[torch.Tensor] = None
```

Comment:
Should be

`Optional[List[torch.tensor]]`

Reply:
Done.